Gemfile.lock

```diff
@@ -315,11 +315,11 @@ GEM
     thread_safe (0.3.6)
     tilt (1.4.1)
     timecop (0.9.1)
-    tins (1.15.1)
+    tins (1.16.3)
     tool (0.2.3)
     tzinfo (1.2.4)
       thread_safe (~> 0.1)
-    uglifier (3.2.0)
+    uglifier (4.0.2)
```

Comment:
@jywarren any idea why it was bumped up by a major version? Wouldn't `4.0.2` be inconsistent with `3.2.0`?